Client/Frontend/Autofill/CreditCard/CreditCardSettingsView/SingleCardView.swift

```diff
@@ -0,0 +1,241 @@
+        contentView.addSubviews(cardTableView, buttonsContainerStackView)
+        view.addSubview(contentView)
+
+        let headerHeight = SingleCardViewControllerUX.headerPrefferredHeight
+        let estimatedCellHeight = cardTableView.estimatedRowHeight
+        let estimatedTableHeight = headerHeight + estimatedCellHeight + SingleCardViewControllerUX.distanceBetweenHeaderAndCells + SingleCardViewControllerUX.distanceBetweenButtonAndTable
+
+        print("estimatedTableHeight = \(estimatedTableHeight)")
+        tableViewHeightConstraint = cardTableView.heightAnchor.constraint(equalToConstant: estimatedTableHeight)
+        tableViewHeightConstraint.priority = UILayoutPriority(999)
+
+        var contentViewHeight = SingleCardViewControllerUX.distanceBetweenHeaderAndTop + estimatedTableHeight + SingleCardViewControllerUX.yesButtonHeight + SingleCardViewControllerUX.bottomSpacing
+        if isIPAD { contentViewHeight += SingleCardViewControllerUX.yesButtonHeight } // one more button on iPad
+        print("contentViewHeight = \(contentViewHeight)")
+        NSLayoutConstraint.activate([
+            contentView.topAnchor.constraint(equalTo: view.topAnchor),
+            contentView.bottomAnchor.constraint(equalTo: view.bottomAnchor),
+            contentView.widthAnchor.constraint(equalToConstant: SingleCardViewControllerUX.contentViewWidth),
+            contentView.heightAnchor.constraint(equalToConstant: contentViewHeight),
+            contentView.centerXAnchor.constraint(equalTo: view.centerXAnchor),
+
+            cardTableView.topAnchor.constraint(equalTo: contentView.topAnchor, constant: SingleCardViewControllerUX.distanceBetweenHeaderAndTop),
+            cardTableView.bottomAnchor.constraint(equalTo: buttonsContainerStackView.topAnchor, constant: 0),
+            cardTableView.leadingAnchor.constraint(equalTo: contentView.leadingAnchor, constant: SingleCardViewControllerUX.tableMargin),
+            cardTableView.trailingAnchor.constraint(equalTo: contentView.trailingAnchor, constant: -SingleCardViewControllerUX.tableMargin),
+
+            buttonsContainerStackView.bottomAnchor.constraint(equalTo: contentView.bottomAnchor, constant: -SingleCardViewControllerUX.bottomSpacing),
+            buttonsContainerStackView.leadingAnchor.constraint(equalTo: contentView.leadingAnchor, constant: SingleCardViewControllerUX.tableMargin),
+            buttonsContainerStackView.trailingAnchor.constraint(equalTo: contentView.trailingAnchor, constant: -SingleCardViewControllerUX.tableMargin),
+
+            yesButton.heightAnchor.constraint(equalToConstant: SingleCardViewControllerUX.yesButtonHeight),
+            notNowButton.heightAnchor.constraint(equalToConstant: SingleCardViewControllerUX.yesButtonHeight),
+            tableViewHeightConstraint
+        ])
+    }
+
+    @objc
+    func didTapYes() {
+        print("didTapYes")
+    }
+
+    @objc
+    func didTapNotNow() {
```

Comment:
nit: can we make it private if its not used anywhere else
```suggestion
    private func didTapNotNow() {
```